samples/languages/python/README.md

```diff
@@ -1,20 +1,43 @@
 SGX-LKL-OE Python and NumPy Sample Application
 ==============================================
 
-1. First build the Python and NumPy file system image:
+In this sample you will run a simple Python application in SGX-LKL and get used to the available tools.
+The aim of this sample is to verify that an application works in SGX-LKL without considering more advanced topics like encryption, attestation, networking, and deployment via Docker.
 
-```
-make
+The Python application is packaged as a Docker image. Let's first verify that it works outside of SGX-LKL:
+
+```sh
+docker build -t pythonapp .
+docker run --rm pythonapp
+# Output:
+# Confidential Computing using Intel SGX in Python with NumPy... 
+# [[   0    1    2 ...   97   98   99]
+#  [ 100  101  102 ...  197  198  199]
+#  [ 200  201  202 ...  297  298  299]
+#  ...
+#  [9700 9701 9702 ... 9797 9798 9799]
+#  [9800 9801 9802 ... 9897 9898 9899]
+#  [9900 9901 9902 ... 9997 9998 9999]]
 ```
 
-2. Run the example application using:
+The Python script is located in the `app/` folder, the Docker image is defined in the `Dockerfile`. Have a look at both files to understand what is happening.
 
-```
-make run-hw
+We can now convert the Docker image into a filesystem image which SGX-LKL can use:
```

Comment:
Is this a typo? "which" -> "with"?

Reply:
No, alternatively "that SGX-LKL can use". ... or something else altogether, like "We can now convert the Docker image into a filesystem image suitable for SGX-LKL:". Which one do you prefer?

Reply:
Changed it to the latter.